src/systemtest/resources/jenkinsJobTemplate.xml

```diff
@@ -12,9 +27,10 @@
   <triggers/>
   <concurrentBuild>false</concurrentBuild>
   <builders>
-    <hudson.tasks.Shell>
-      <command>${shell.script}</command>
-    </hudson.tasks.Shell>
+    <com.ericsson.eiffel.fem.buildstep.GroovyBuildStep plugin="eiffel-fem@0.0.86">
```

Comment:
Why using Fem Plugin?,,, This should not test FEM functionalities.
Tests must work with both Jenkins without FEM plugin and with Fem plugin.
Use normal Groovy build step instead.

Reply:
This will be removed when we switch for the jenkins manager in commons library. There we use the normal groovy plugin

Reply:
Yes, i should have changed that. When i use the new jenkinsmanager from the commons library this will be fixed. I can change that now as well.

Reply:
ok